docs/source/contributing.md

```diff
@@ -1,5 +1,7 @@
 # Contributing
 
+The ipywidgets package is under the [Jupyter-Widgets](https://github.com/jupyter-widgets) [software subproject](https://jupyter.org/governance/software_subprojects.html).
```

Comment:
I thought that this was the best place to describe the hierarchy for the official docs site.  It could also go in `docs/source/index.md` .  Open to suggestions.